docs/installingNative.md

```diff
@@ -71,22 +71,11 @@ Now you can install the other dependencies.
 
 === "macOS"
 
-    Brew can install the rest of the dependencies.
+    Brew can install the rest of the dependencies. Enabling `corepack` makes `yarn` available.
```

Comment:
It's not clear to me why we separate things into "prerequisites" and "rest of the dependencies", especially since the bulleted list is labeled "prerequisites" but also contains the "other dependencies" (Node). Consider merging them.

Reply:
Combined in abbf17d9086180bf6b102f050e23f7e070d66274